lang/levels/standard/text_fre.po

```diff
@@ -0,0 +1,841 @@
+
+#. The Death Clock (lv450)
+#: guitext:118
+msgctxt "In-game message"
+msgid "In the east the heroes have a fortress. Why don't you go find out what it contains?"
+msgstr "A l'est les héros ont une forteresse. Pourquoi n'irais-tu pas découvrir ce qu'il contient ?"
+
+#. The Death Clock (lv450)
+#: guitext:119
+msgctxt "In-game message"
+msgid "Mole-man save us, they are almost here! Place more traps now or hope your little army is stronger than it looks."
+msgstr "Taupe-man sauve-nous, ils sont presque là ! Placez plus de pièges maintenant ou espérez que votre petite armée est plus forte qu'elle n'en a l'air."
+
+#. The Death Clock (lv450)
+#: guitext:120
+msgctxt "In-game message"
+msgid "I have a bad feeling about this. You know you are supposed to kill the little white men right?"
+msgstr "J'ai un mauvais pressentiment à propos de ceci. Vous savez que vous êtes censé tuer les petits hommes blancs, n'est-ce pas ?"
+
+#. The Death Clock (lv450)
+#: guitext:121
+msgctxt "In-game message"
+msgid "Yes, we have some creatures left! I never doubted you for a second. Congratulations!"
+msgstr "Oui, il nous reste quelques créatures ! Je n'ai jamais douté de vous une seconde. Félicitations!"
+
+#. The Death Clock (lv450)
+#: guitext:122
+msgctxt "In-game message"
+msgid "You lost your only creature. You needed it to get claim the tiles around your heart. Hopeless."
+msgstr "Vous avez perdu votre unique créature. Vous en aviez besoin pour réclamer les tuiles autour de votre coeur. Désespéré."
+
+#: guitext:123 guitext:124 guitext:125 guitext:126 guitext:127 guitext:128
+#: guitext:129 guitext:130 guitext:131 guitext:132 guitext:133 guitext:134
+#: guitext:135 guitext:136 guitext:137 guitext:138 guitext:139 guitext:140
+#: guitext:141 guitext:142 guitext:143 guitext:144 guitext:145 guitext:146
+#: guitext:147 guitext:148 guitext:149 guitext:150 guitext:151 guitext:152
+#: guitext:153 guitext:154 guitext:155 guitext:156 guitext:157 guitext:158
+#: guitext:159 guitext:160 guitext:161 guitext:162 guitext:163 guitext:164
+#: guitext:165 guitext:166 guitext:167 guitext:168 guitext:169 guitext:170
+#: guitext:171 guitext:172 guitext:173 guitext:174 guitext:175 guitext:176
+#: guitext:177 guitext:178 guitext:179 guitext:180 guitext:181 guitext:182
+#: guitext:183 guitext:184 guitext:185 guitext:186 guitext:187 guitext:188
+#: guitext:189 guitext:190 guitext:191 guitext:192 guitext:193 guitext:194
+#: guitext:195 guitext:196 guitext:197 guitext:198 guitext:199 guitext:200
+msgctxt "In-game message"
+msgid "Moo"
+msgstr "Meuuh"
+
+#: guitext:201
+msgctxt "Level name"
+msgid "Tymnath Herez"
+msgstr "Tymnath Herez"
+
+#: guitext:202
+msgctxt "Level name"
+msgid "Yami"
+msgstr "Yami"
+
+#: guitext:203
+msgctxt "Level name"
+msgid "Temple Deal"
+msgstr "Temple Deal"
+
+#: guitext:204
+msgctxt "Level name"
+msgid "Grail Run"
+msgstr "Grail Run"
+
+#: guitext:205
+msgctxt "Level name"
+msgid "Thug of War"
+msgstr "Thug of War"
```

Comment:
Fou de guerre? ou Voyou de guerre?

Reply:
It's a pun of the rope game "Tug of war" and the "Thug" like criminal. I used "Tug of war" in German because loobinex told me the gameplay was like this.

Reply:
@CaptainCosmotic Thanks! 
"Tir à la corde" seems to be the appropriate translation for this one?